pkg/features/features.go

```diff
@@ -85,6 +85,12 @@ const (
 	// owner: @rrhubenov
 	// alpha: v1.124.0
 	OpenTelemetryCollector featuregate.Feature = "OpenTelemetryCollector"
+
+	// VpaInPlaceOrRecreateUpdateMode enables the usage of in-place Pod resource updates in the Vertical Pod Autoscaler deployments.
+	// resources to perform in-place Pod resource updates.
+	// owner: @vitanovs
```

Comment:
You could also add me as 2nd feature gate owner. :) 

```suggestion
	// owner: @vitanovs @ialidzhikov
```